### ISSUE TITLE
pip wheels for Windows

### ISSUE THREAD

**Issue by benjaminweb** [scoder/quicktions#4]
**Problem**: ``pip3 install quicktions`` requires [Microsoft Visual C++ Build Tools](http://landinghub.visualstudio.com/visual-cpp-build-tools) to compile (it claims to require 4 GiB of space, let alone the time constraint).

**Proposal**: Provide pip wheels (maybe through appveyor?) to lift the dependency, hence ease distribution of quicktions.

```
 ...
  Running setup.py bdist_wheel for quicktions ... error
  Complete output from command c:\users\ieuser\appdata\local\programs\python\python36\python.exe -u -c "import setuptool
s, tokenize;__file__='C:\\Users\\IEUser\\AppData\\Local\\Temp\\pip-build-wkidjm4r\\quicktions\\setup.py';f=getattr(token
ize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_
wheel -d C:\Users\IEUser\AppData\Local\Temp\tmpubzpb6dgpip-wheel- --python-tag cp36:
  c:\users\ieuser\appdata\local\programs\python\python36\lib\distutils\dist.py:261: UserWarning: Unknown distribution op
tion: 'bugtrack_url'
    warnings.warn(msg)
  running bdist_wheel
  running build
  running build_ext
  building 'quicktions' extension
  error: Microsoft Visual C++ 14.0 is required. Get it with "Microsoft Visual C++ Build Tools": http://landinghub.visualstudio.com/visual-cpp-build-tools

  ----------------------------------------
```

**Comment by benjaminweb** [comment]
I've set up appveyor and travis configs to create builds.
Though, I needed to replace distutils with setuptools in ``setup.py`` to let ``setup.py`` produce wheels.

If you want to adopt the CI configs:

https://github.com/benjaminweb/quicktions/blob/master/.travis.yml
https://github.com/benjaminweb/quicktions/blob/master/appveyor.yml

Added files: ``appveyor_env.cmd``, ``setup.cfg`` (for wheel; also added to ``MANIFEST.in``)

Corresponding passing builds:
https://travis-ci.org/benjaminweb/quicktions
https://ci.appveyor.com/project/hyllos/quicktions

Both allow automatic upload of wheels to pypi for tagged releases (untested by me).
All required would be setting the TWINE_PASSWORD ENV in your travis and/or appveyor profile.
